infra/recipes/terraform/azure/environments/container/main.tf

```diff
@@ -0,0 +1,8 @@
+resource "azurerm_resource_group" "openhouse_sandbox" {
+    name = var.resource_group_name
+    location = var.resource_group_location
+}
+
+module "container" {
+    source = "../../modules/container"
+}
```

Comment:
Is it possible to define a output.tf to publish output container_registry information thats being referenced in sandbox?

Reply:
An output.tf is not necessary I believe because we can import the container_registry data using the data block in sandbox. The module here just allows for future reusal of code.